scripts/i18n/i18n_update.py

```diff
@@ -24,20 +24,23 @@
     )
 )
 
-from common.i18n import (
-    DEFAULT_LOCALE,
-    locale_localized_name,
-    locale_flag_url,
-    translators,
-)
-from common.logger import (
-    print_interactive_error,
-    print_interactive_warning,
-    print_interactive_info,
-    print_interactive_success,
-    input_interactive,
-    print_interactive_input,
-)
+if (
+    True
+):  # to pass pre-commit tests (otherwise E402 Module level import not at top of file)
+    from common.i18n import (
+        DEFAULT_LOCALE,
+        locale_localized_name,
+        locale_flag_url,
+        translators,
+    )
+    from common.logger import (
+        print_interactive_error,
+        print_interactive_warning,
+        print_interactive_info,
+        print_interactive_success,
+        input_interactive,
+        print_interactive_input,
+    )
```

Comment:
```suggestion
from common.i18n import (  # Noqa: E402
    DEFAULT_LOCALE,
    locale_localized_name,
    locale_flag_url,
    translators,
)
from common.logger import (  # Noqa: E402
    print_interactive_error,
    print_interactive_warning,
    print_interactive_info,
    print_interactive_success,
    input_interactive,
    print_interactive_input,
)
```

There is the `Noqa` comment instruction for that